Dockerfile.reborn

```
@@ -14,6 +14,17 @@
 
 FROM rust:1.92-bookworm AS chef
 
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)